test/Interpreter/Inputs/ObjCClasses/ObjCClasses.m

```diff
@@ -132,3 +133,15 @@ - (NSInteger) count {
   return _objects.count;
 }
 @end
+
+@implementation TestingBool
+
+- (void) shouldBeTrueObjCBool: (BOOL)value {
+  assert(value);
```

Comment:
I think you should make the tests even stronger: the memory representation of `value` should be `1`. These tests would pass today with the -1 issue.

```c
static_assert(sizeof(BOOL) == sizeof(char), "unknown architecture");
char expected = 1;
assert(0 == memcmp(&value, &value, 1));
```